app/helpers/container_summary_helper.rb

```diff
@@ -171,7 +171,8 @@ def textual_persistent_volume
   end
 
   def textual_persistent_volumes
-    textual_link(@record.persistent_volumes)
+    link = url_for_only_path(:id => @record.id, :action => "show", :display => "persistent_volumes")
```

Comment:
did you still need to use `url_for_only_path` now that you taught the system how to deal with `persistent_volumes`? um, did you test if the url "just works"?